bot/exts/filtering/_ui/ui.py

```diff
@@ -617,6 +618,9 @@ class AlertView(discord.ui.View):
     def __init__(self, ctx: FilterContext, triggered_filters: dict[FilterList, list[str]] | None = None):
         super().__init__(timeout=ALERT_VIEW_TIMEOUT)
         self.ctx = ctx
+        if "banned" in self.ctx.action_descriptions:
+            # If the user has already been banned, do not attempt to add phishing button
```

Comment:
Is this added because if they were autobanned then it's probably because the URL they sent is already added as a filter? If so, I think the comment should explain that.

Reply:
```suggestion
            # If the user has already been banned, do not attempt to add phishing button since the URL or guild invite
            # is probably already added as a filter
```
something like this?

Reply:
that looks good to me 👍

Reply:
https://github.com/python-discord/bot/pull/2982/commits/dce89a9ab3be38a4f88fab449a7158c6c75989fc